utils.c

```diff
@@ -558,3 +558,58 @@ int janus_pidfile_remove(void) {
 	g_free(pidfile);
 	return 0;
 }
+
+void janus_get_json_type_name(int jtype, gboolean positive_non_empty, char *type_name) {
+	if(positive_non_empty)
+		strcpy(type_name,
```

Comment:
Use the Glib alternatives where possible (so ```g_strcpy``` instead of ```strcpy```, ```g_strcat``` instead of ```strcpy```) as they allow for easier portability and are ostensibly safer too.